arviz/plots/khatplot.py

```diff
@@ -302,3 +303,8 @@ def hover(event):
                     fig.canvas.draw_idle()
 
     fig.canvas.mpl_connect("motion_notify_event", hover)
+
+
+@conditional_jit
+def _khat_histogram(data, bin_edges):
```

Comment:
The numba 0.44 update has thrown up a lot of significant changes and they are planning to remove some functions by 0.46. So I have kept these functions separate to ease the debugging in case something breaks.